### DISCUSSION TITLE
./docker.sh restore path-to-backup.zip doesn't work

### DISCUSSION THREAD

**Discussion by Mozitux** [AzuraCast/AzuraCast#5649] [Q&A] [answered]
Hello, I try to restore an Azuracast backup on my Debian 11 Server using Docker and I have the following error message:
`./docker.sh: No such file or directory`

Docker is well installed, but I don't know why I have this message...
How can I proceed to restore my backup?

**Comment by ItsMitchh** [comment, marked as answer]
Are you inside the /var/azuracast directory?

**Reply by Mozitux** [reply]
My bad
